src/Components/Radio.tsx

```diff
@@ -0,0 +1,25 @@
+import type { JSX } from 'preact';
+
+type RadioProps = {
+    id: string;
+    label: string;
+    radioVariable: string;
+    value: string;
+    name: string;
+    onChange: JSX.EventHandler<JSX.TargetedEvent<HTMLInputElement, Event>>;
```

Comment:
@zner0L I'd like these to be passed `e.target.value` directly, the DOM event seems like an irrelevant implementation detail (and it needs annoying TS casting on the using side). But that would require quite extensive changes, so I'll hold off on that for now.